linker_scripts/audio_table_rodata.ld

```diff
@@ -0,0 +1,17 @@
+OUTPUT_ARCH (mips)
```

Comment:
With this in linker_scripts it is actually getting used in the final link as well which is probably not ideal.
Not sure if we should move it to a different folder or add a subfolder that gets excluded from the find command?
```bash
mips-linux-gnu-ld -T build/n64-us/mm-n64-us.ld -T build/n64-us/linker_scripts/audio_table_rodata.ld build/n64-us/linker_scripts/extra.ld build/n64-us/linker_scripts/hardware_regs.ld build/n64-us/linker_scripts/undefined_syms.ld --no-check-sections --accept-unknown-input-arch --emit-relocs -Map build/n64-us/mm-n64-us.map -o build/n64-us/mm-n64-us.elf
```